docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -234,3 +234,12 @@ Important events are also reported through Kubernetes events, such as when two c
 ----
 17s    Warning   ReconciliationError stackconfigpolicy/config-test   StackConfigPolicy is an enterprise feature. Enterprise features are disabled
 ----
+
+[float]
+[id="{p}-{page_id}-specifics-snap-repo"]
+== Specifics for snapshot repositories
+
+In order to avoid a conflict between multiple Elasticsearch clusters writing their snapshot to the same location, ECK automatically:
+- sets the `base_path` to `snapshots/<namespace><esName>` when it is not provided, for Azure, GCS and S3 repositories
+- appends `<namespace><esName>` to `location` for a FS repository
+- appends `<namespace><esName>` to `path` for an `HDFS` repository
```

Comment:
* Something is wrong in the final rendering, I think a `\n` is missing before the list.
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/976373/232982420-6ae8016a-d877-4705-9e62-f9f9b505c5fd.png">

* I think the hyphen is missing as well (`uniqSuffix := fmt.Sprintf("%s-%s", esNs, esName)`)?

* Should we make `snapshot` the word plural?

```suggestion
In order to avoid a conflict between multiple Elasticsearch clusters writing their snapshots to the same location, ECK automatically:

- sets the `base_path` to `snapshots/<namespace>-<esName>` when it is not provided, for Azure, GCS and S3 repositories.
- appends `<namespace>-<esName>` to `location` for a FS repository.
- appends `<namespace>-<esName>` to `path` for an `HDFS` repository.
```

Reply:
Triple-yes. 🙇